karma.conf.js

```diff
@@ -1,28 +1,47 @@
+const webpack = require('webpack');
+
 module.exports = (config) => {
   config.set({
-    frameworks: ['mocha', 'chai'],
+    frameworks: ['mocha', 'chai', 'webpack'],
     files: ['./index.js', './test.spec.js'],
     preprocessors: {
       './index.js': ['webpack'],
       './test.spec.js': ['webpack'],
     },
     webpack: {
-      node: {
-        fs: 'empty',
+      resolve: {
+        fallback: {
+          fs: false,
+          crypto: require.resolve('crypto-browserify'),
+          buffer: require.resolve('buffer/'),
+          assert: require.resolve('assert-browserify'),
+          stream: require.resolve('stream-browserify'),
+          path: require.resolve('path-browserify'),
+          url: require.resolve('url/'),
+        }
       },
+      plugins: [
+        new webpack.ProvidePlugin({
+          Buffer: ["buffer", "Buffer"],
+          process: 'process/browser',
+        })
+      ],
       module: {
         rules: [
           { test: /\.dat$/, use: 'raw-loader' },
-          { enforce: 'post', loader: 'transform-loader?brfs' },
+          { enforce: 'post', loader: "transform-loader", options: "brfs-node-15" },
         ],
       },
     },
     reporters: ['mocha'],
     port: 9876,
     colors: true,
     autoWatch: false,
-    browsers: ['ChromeHeadless', 'FirefoxHeadless'],
+    browsers: ['ChromeHeadless'],
```

Comment:
You guys decided to keep only Chrome?

Reply:
Not at all, something got wrong !

Reply:
That is put back, thank you for noticing, I guess wrongly commited the whole file during the ChromeHeadless fixes >.>'